workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -0,0 +1,723 @@
+                            "examples": ["0x0000000000000000000000000000000000001010"]
+                        },
+                        "addressType": {
+                            "$comment": "{\"term\": \"address\", \"@id\": \"https://schema.org/Intangible\"}",
+                            "title": "addressType",
+                            "description": "AddressType of the token smart contract.",
+                            "type": "string",
+                            "examples": ["MaticNameSpace"]
+                        },
+                        "addressAlg": {
+                            "$comment": "{\"term\": \"addressAlg\", \"@id\": \"https://schema.org/algorithm\"}",
+                            "title": "addressAlg",
+                            "description": "Algorithm used to create the address e.g. CREATE2 or the standard ethereum address construction which is the last 40 characters/20 bytes of the Keccak-256 hash of a secp256k1 public key.",
+                            "type": "string",
+                            "examples": ["CREATE2"]
+                        },
+                        "tokenName": {
+                            "$comment": "{\"term\": \"tokenName\", \"@id\": \"https://schema.org/name\"}",
+                            "title": "tokenName",
+                            "description": "Token name.",
+                            "type": "string",
+                            "examples": ["Matic"]
+                        },
+                        "tokenSymbol": {
+                            "$comment": "{\"term\": \"tokenSymbol\", \"@id\": \"https://schema.org/currency\"}",
+                            "title": "tokenSymbol",
+                            "description": "Token symbol e.g. ETH.",
+                            "type": "string",
+                            "examples": ["MATIC"]
+                        },
+                        "humanReadableTokenSymbol": {
+                            "$comment": "{\"term\": \"humanReadableTokenSymbol\", \"@id\": \"https://schema.org/currency\"}",
+                            "title": "humanReadableTokenSymbol",
+                            "description": "A Token symbol e.g. ETH, concatenated with the `chainId` the token was issued on or bridged to, e.g. ETH-1",
+                            "type": "string",
+                            "examples": ["MATIC-137"]
+                        },
+                        "decimals": {
+                            "$comment": "{\"term\": \"decimals\", \"@id\": \"https://schema.org/Number\"}",
+                            "title": "decimals",
+                            "description": "Allowed number of decimals for the listed token. This property may be named differently by token standards e.g. granularity for ERC-777",
+                            "type": "integer",
+                            "examples": [18]
+                        },
+                        "tokenLogoURI": {
```

Comment:
Tokenlist uses `logoURI` instead of `tokenLogoURI`

Reply:
reverted